kivymd/uix/button/button.py

```diff
@@ -86,14 +86,15 @@ def build(self):
 By default, the color of :class:`~MDIconButton`
 (depending on the style of the application) is black or white.
 You can change the color of :class:`~MDIconButton` as the text color
-of :class:`~kivymd.uix.label.MDLabel`:
+of :class:`~kivymd.uix.label.MDLabel`, substituting ``theme_icon_color`` for
+``theme_text_color`` and ``icon_color`` for ``text_color``.
 
 .. code-block:: kv
 
     MDIconButton:
         icon: "android"
-        theme_text_color: "Custom"
-        text_color: app.theme_cls.primary_color
```

Comment:
This is the API change. Therefore, we have to specify the corresponding property (deprecated) - deprecated=True

We can't just take and delete a property that users are using. We must warn users that this property will be removed in the future.